Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -11400,6 +11400,18 @@ bool CStaticFunctionDefinitions::GetAccountSerial(CAccount* pAccount, SString& s
     return bRegistered;
 }
 
+
+bool CStaticFunctionDefinitions::SetAccountSerial(const std::string& name, const std::string& serial) noexcept
```

Comment:
I think it would be better to use `CAccount` since the other functions use it as well
```suggestion
bool CStaticFunctionDefinitions::SetAccountSerial(CAccount* account, const std::string& serial) noexcept
```

Reply:
adjusted 790ffa2